### ISSUE TITLE
Error when saving a matrix into csv file

### ISSUE THREAD

**Issue by dphilippon** [gama-platform/gama#2216]
### Steps to reproduce
Run the following model 
````
model saveStatement

/* Insert your model definition here */

global
{
	matrix t <- matrix([[1,2,3],[2,3,4],[3,4,5]]);
	init
	{
		write t;
		do save_matrix(t,"./example.csv");
	}
	action save_matrix (matrix x, string f)
	{
		save x to: f type:csv rewrite:true header:true;
	}
}
experiment test type:gui;
````
### Expected behavior
The saved file should be in a good format (ie :  rows and columns should be like the matrix)

### Actual behavior
The matrix is saved like an one line list

**Comment by dphilippon** [comment]
I just commit something to correct that,if any problem arises, feel free to open this issue.

**Comment by AlexisDrogoul** [comment]
You're right; it is supposed to have been fixed in cd2b4a03182718be8ed4d3d2d90551eec4383806.

I will check. Thanks for reporting.
